server/data/vocabularies.json

```diff
@@ -261,19 +261,19 @@
         "selection_type": "single selection",
```

Comment:
Need to increment `init_version` attribute, otherwise the changes won't be applied automatically on deploy

Reply:
yep